src/Chapter1.hs

```diff
@@ -503,7 +505,8 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y = if (abs x <= abs y) then x
+                                       else y
```

Comment:
Right! Your solution is correct 💯 

A few style suggestions (which you can ignore, I just share it as the additional info 😄 ). You don't need to take the expression after `if` in parenthesis.
Also, people usually align if-then-else either in one line, or each clause at the separate line with the same indentation:

```haskell
foo = if x then y else z

foo = 
    if x
    then y
    else z
```